triage/nw_prj_cli.py

```
@@ -0,0 +1,295 @@
+            v = cfg.get(k)
+            if v:
+                rp = _resolve(v, base)
+                params[k] = str(rp) if rp else None
+        for k in ("months", "webexcel", "zip_output", "run_date", "protected_names"):
+            if k in cfg:
+                params[k] = cfg[k]
+
+    # CLI flags override config
+    for k in ("roster_log", "admin_folder", "admin_april", "admin_may", "out_dir", "run_date"):
+        cli_val = getattr(args, k.replace("admin_april", "admin_april").replace("admin_may", "admin_may"), None)
+        if cli_val and (k not in params or k == "out_dir" and params.get(k) is None):
```

Ruff (0.15.15)

[warning] Parenthesize `a and b` expressions when chaining `and` and `or` together, to make the precedence clear

Parenthesize the `and` subexpression

(RUF021)